config/__init__.py

```diff
@@ -134,11 +135,16 @@ def update_status_config(milestones_content):
 # it will create a line with the following format
 # 2015-09-14 20:50:19,185 INFO: Thing_To_Log [in /codepath/views.py:127]
 
-LOG_FILE = '/tmp/webcompat.log'
+# set the tempdir to somewhere predictable, no matter the platform
+tempfile.tempdir = os.path.join(os.getcwd(), 'tmp')
+print('Writing logs to: {}'.format(tempfile.gettempdir()))
+
+LOG_FILE = os.path.join(tempfile.gettempdir(), 'webcompat.log')
```

Comment:
ok let's see

```python
>>> import os
>>> import tempfile
>>> tempfile.tempdir = os.path.join(os.getcwd(), 'tmp')
>>> tempfile.gettempdir()
'/Users/karl/code/webcompat.com/tmp'
>>> os.path.join(tempfile.gettempdir(), 'webcompat.log')
'/Users/karl/code/webcompat.com/tmp/webcompat.log'
```

Just double checking, we have 
https://github.com/webcompat/webcompat.com/blob/0a0a212373ddcaaaf39dab502486753588c6336a/.gitignore#L16

And 

```
http HEAD http://localhost:5000/tmp/webcompat.log
```

```http
HTTP/1.0 404 NOT FOUND
Content-Length: 38066
Content-Security-Policy: default-src 'self'; object-src 'none'; connect-src 'self' https://api.github.com; font-src 'self' https://fonts.gstatic.com; img-src 'self' https://webcompat.com https://www.google-analytics.com https://*.githubusercontent.com data:; manifest-src 'self'; script-src 'self' https://www.google-analytics.com https://api.github.com; style-src 'self' 'unsafe-inline' https://fonts.googleapis.com; report-uri /csp-report
Content-Type: text/html; charset=utf-8
Date: Mon, 18 Jun 2018 20:27:35 GMT
Server: Werkzeug/0.14.1 Python/2.7.10
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-XSS-Protection: 1; mode=block
```

So far so good.